app/components/wokes/authors_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<section>
+  <header aria-describedby="popover-work.author">
+    Authors to include in citation
+    <%= render PopoverComponent.new key: "work.author" %>
+  </header>
+  <p>When there are multiple authors, list them in the order they should appear in the citation. If you need to change the order of the authors, click the arrows to move individual authors up or down in the list.</p>
+
+  <%= render NestedFormComponent.new(form: form, controller: "author-form", row: Wokes::ContributorRowComponent,
```

Comment:
Authors are just like contributors except they use a different controller name, a different field ("authors" instead of "contributors"), a different form class, and are ordered. However, the row component is the same (which is where the bulk of the rendering is).